server/src/game/index.js

```diff
@@ -99,6 +102,35 @@ export class Game {
     this.log(`will start in ${env.START_GAME_TIMEOUT} seconds`);
   }
 
+  spawnMobs() {
+    const getRandomSpawnPoint = (map) => {
+      let spawnPoint;
+
+      while (!spawnPoint) {
+        const randomPoint = [
+          Math.floor(Math.random() * env.MAP_HEIGHT),
+          Math.floor(Math.random() * env.MAP_WIDTH),
+        ];
+        const alreadyTaken = [...this.mobs.values()].some(mob => [mob.x, mob.y] === randomPoint);
+        if (!alreadyTaken && map[randomPoint[0]][randomPoint[1]] === ' ') {
+          spawnPoint = randomPoint;
+        }
+      }
+
+      return spawnPoint;
+    };
+
+    MOBS_TYPE.map(type => {
+      let lastId = this.mobs.size > 0 ? parseInt([...this.mobs.keys()][this.mobs.size - 1], 10) + 1 : 0;
+      for (let i = lastId ; i < ( lastId + MOBS_FOR_ONE_PLAYER[type] * this.users.size); i += 1) {
+        const mob = new Mob(i, this, type, getRandomSpawnPoint(this.map));
```

Comment:
we don't need to pass down the `map`, you can access to `this.map` inside of `getRandomSpawnPoint`